x/ibc/02-client/keeper/keeper.go

```diff
@@ -135,8 +135,9 @@ func (k Keeper) GetSelfConsensusState(ctx sdk.Context, height uint64) (exported.
 	valSet := stakingtypes.Validators(histInfo.Valset)
 
 	consensusState := ibctmtypes.ConsensusState{
-		Height:       height,
-		Timestamp:    ctx.BlockTime(),
+		Height: height,
+		// FIXME: Currently commented out due to time normalisation issues.
+		//Timestamp:    histInfo.Header.Time,
```

Comment:
This is going to get fixed in this PR?

Reply:
I wasn't able to figure it out, some sort of normalisation (maybe in Tendermint?).

We should either fix it or open an issue to track it.

Reply:
What feature/guarantee/check are we missing by not having this timestamp?

Reply:
https://github.com/cosmos/cosmos-sdk/issues/5762